doc/changes/latest.inc

```diff
@@ -32,4 +34,4 @@ Bugs
 API changes
 ~~~~~~~~~~~
 
-- ...
+- ``ICA.max_pca_components`` will not be altered by calling `~mne.preprocessing.ICA.fit` anymore. Instead, the new attribute ``ICA.max_pca_components_`` will be set, by `Richard Höchenberger`_ (:gh:`8321`)
```

Comment:
This is a BUG I think actually

Reply:
Yes I was actually pondering between API and Bug, but I thought since we're also introducing a new attribute here, it should go to API… otherwise I would have to add yet another entry for Bug?

Reply:
+1 for BUG fix !

Reply:
Done